src/base/net/reverseresolution.cpp

```diff
@@ -65,7 +65,7 @@ void ReverseResolution::resolve(const QString &ip)
     }
     else {
         // Actually resolve the ip
-        m_lookups.insert(QHostInfo::lookupHost(ip, this, SLOT(hostResolved(QHostInfo))), ip);
+        m_lookups.insert(QHostInfo::lookupHost(ip, this, &ReverseResolution::hostResolved), ip);
```

Comment:
This one is only available on Qt >= 5.9, you can make it so:
```c++
#if (QT_VERSION >= QT_VERSION_CHECK(5, 9, 0))
m_lookups.insert(QHostInfo::lookupHost(ip, this, &ReverseResolution::hostResolved), ip);
#else
m_lookups.insert(QHostInfo::lookupHost(ip, this, SLOT(hostResolved(QHostInfo))), ip);
#endif
```

Reply:
Thanks! I didn't even look if the checks failed :(